pkg/storage/pvc/pvc.go

```diff
@@ -24,7 +24,10 @@ import (
 const (
 	rootDirectory      = "/registry"
 	randomSecretSize   = 32
-	pvcOwnerAnnotation = "imageregistry.openshift.io"
+	PVCOwnerAnnotation = "imageregistry.openshift.io"
+
+	// PVCImageRegistryName is the defailt name of the claim provisioned for PVC backend
+	PVCImageRegistryName = defaults.ImageRegistryName + "-storage"
```

Comment:
Let's move this constant to `defaults`. And in that package it's better to have something like `PVCImageRegistryName = "image-registry-storage"` to make it greppable.

Reply:
makes sense! done